SlnfUpdater/FileStructure/SlnfJsonStructured.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Pastel;
+using SlnfUpdater.Helper;
+using System.Text.RegularExpressions;
+
+namespace SlnfUpdater.FileStructure
+{
+    public sealed class SlnfJsonStructured
```

Comment:
@GRAADD I'm not sure if this name is OK. Compare: SlnfJson - SlnfJsonStructured - SlnfSolutionJson. It is not very unique names. Could you rename this class back to `Slnf`?